core/dataURI/dataURI.go

```diff
@@ -206,7 +206,7 @@ func GetData(uri string, isTempData bool) (io.Reader, common.SyncServiceError) {
 		trace.Trace("Retrieving data from %s", filePath)
 	}
 
-	file, err := os.Open(filePath)
+	file, err := os.Open(filepath.Clean(filePath))
```

Comment:
I don't think this change is necessary, the `filePath` is already cleaned, see line 196
```
filePath, err := filepath.Abs(filepath.Clean(dataURI.Path))
```